search/search_repository.go

```diff
@@ -251,8 +260,8 @@ func generateSQLSearchInfo(keywords searchKeyword) (sqlParameter string) {
 
 // extracted this function from List() in order to close the rows object with "defer" for more readability
 // workaround for https://github.com/lib/pq/issues/81
-func (r *GormSearchRepository) search(ctx context.Context, sqlSearchQueryParameter string, start *int, limit *int) ([]models.WorkItem, uint64, error) {
-	db := r.db.Model(&models.WorkItem{}).Where("tsv @@ query")
+func (r *GormSearchRepository) search(ctx context.Context, sqlSearchQueryParameter string, types []string, start *int, limit *int) ([]models.WorkItem, uint64, error) {
+	db := r.db.Debug().Table("work_items w").Where("tsv @@ query")
```

Comment:
I think it is better to overwrite the `TableName` method in the `GormSearchRepository` to be sure that name is used consistently:

```go
// TableName overrides the table name settings in Gorm to force a specific table name
// in the database.
func (r *GormSearchRepository) TableName() string {
	return "work_items"
}
```

Then use it like this:

```go
db := r.db.Debug().Table(r.TableName() + " w").Where("tsv @@ query")
```